### ISSUE TITLE
email_not_verified prop persistent after password-reset

### ISSUE THREAD

**Issue by JuliaBD** [beepnl/BEEP#85]
Situation:
1. user account with email_not_verified (created earlier, just never clicked email verification link)
2. returns email_not_verified 400 response when trying to login
3. resend verification link with /email/resend POST request
**4. password-reset seems successful (200 OK response)
5. but upon login email_not_verified 400 response is returned, again...**
6. back to step 3, which then returns 'too many attempts' 429 response
7. after a while: steps 1-6 on repeat.

In short: email_not_verified is never set to email_verified_at, should be done with password-reset (/user/reset POST request).

Issue occurs in both v2 and v3.

**Comment by JuliaBD** [comment]
Seems to work ok now!
